src/instrument.js

```diff
@@ -79,19 +115,51 @@ async function createInstrument(instrumentCreatorAccountName, instrumentOwnerAcc
 
 async function getApiCallStats(instrumentId, rightName){
   //calls the usagelog contract to get the total number of calls against a particular right
-  let result = await this.eos.getAllTableRows({
+  let result = await this.eos.getTableRows({
     code: INSTR_USAGE_CONTRACT_NAME,
-    table: LOG_TABLE_NAME,
-    scope: instrumentId // new scoping
+    json: true,
+    scope: instrumentId,
+    table: LOG_COUNT_TABLE_NAME,
+    limit: -1
   })
-  for (var i = 0; i < result.rows.length; i++) {
+
+  for (let i = 0; i < result.rows.length; i++) {
     if ( result.rows[i]["right_name"] === rightName) {
-      const rightProprties = {"totalCalls": calls[i]["total_count"], "totalCpuUsage": calls[i]["total_cpu"]}
+      const rightProprties = {"totalApiCalls": result.rows[i]["total_count"], "totalCpuUsage": result.rows[i]["total_cpu"]}
       return rightProprties
     }
   }
 }
 
+async function getRightStats(rightName, owner){
+  // Returns the total cpu and api calls against a particular right across all the vouchers. If owner specified, then returns the toatal api calls and cpu usage for the owner.
+  let instruments
```

Comment:
If we're not assigning specific values, you can initialize on one line... `let instruments, instrumentList, rightProperties`